examples/graphql-mutations/graphql_mutations.md

```diff
@@ -2,6 +2,8 @@
 
 The Ballerina `graphql` module allows defining GraphQL `Mutation` operations. A `remote` method inside a `graphql:Service` represents a field in the root `Mutation` object type. Therefore, if a `remote` method is present inside the `graphql:Service`, the auto-generated schema will have the `Mutation` type. Each `remote` method in the service will be added as a field of the `Mutation` type. The field name will be the `remote` method name and the field type will be the return type of the `remote` method. Use the `Mutation` operation when performing any side-effects on the underlying data system.
 
+**Note:** GraphQL mutations are expected to perform side-effects on the underlying data system, i.e. mutations are expected to mutate the state of the backend. In Ballerina, `remote` methods are used to handle such cases. Therefore, `remote` methods are used to represent the fields of GraphQL `Mutation` type.
```

Comment:
How about?

GraphQL mutations are actions that are expected to mutate the state of the server. Ballerina uses `remote` methods to handle such cases. Therefore, these remote methods are usually named using verbs.